frontend/public/locales/en/translation.json

```diff
@@ -2295,6 +2295,7 @@
     "template.templates.title": "Ansible job templates",
     "template.title": "Automation",
     "Tenant ID": "Tenant ID",
+    "Testing Modal": "Testing Modal",
```

Comment:
Demo plugin shouldn't be affecting the real app. We need not use translations. If we do want to have translations they should belong to a different namespace altogether.